src/Components/AdvancedSearch/AdvancedSearch.types.ts

```diff
@@ -17,17 +21,18 @@ export interface IAdvancedSearchProps {
         IAdvancedSearchStyleProps,
         IAdvancedSearchStyles
     >;
+    theme: Theme;
```

Comment:
I think we shouldn't need theme here

Reply:
I had it to be able to resolve `var( )` css values but now I don't need it. good catch